push-images.sh

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env bash
 # Do not edit by hand; please use build scripts/templates to make changes
-docker push cimg/node:16.16
-docker push cimg/node:16.16.0
-docker tag cimg/node:16.16.0 cimg/node:lts
-docker push cimg/node:lts
-docker push cimg/node:16.16-browsers
-docker push cimg/node:16.16.0-browsers
-docker tag cimg/node:16.16.0-browsers cimg/node:lts-browsers
-docker push cimg/node:lts-browsers
+docker push cimg/node:18.5
+docker push cimg/node:18.5.0
+docker tag cimg/node:18.5.0 cimg/node:current
```

Comment:
@JalexChen 18.7.0 is the latest current version, are we missing it currently or will this overwrite the tag for 18.7.0?

Reply:
forgot cpe-dash token expired. will add 18.6 and 18.7 to this release after updating the vanilla node install